.github/workflows/build.yaml

```diff
@@ -0,0 +1,122 @@
+name: Build & Test
+
+on: [ push ]
+jobs:
+  tests:
+    name: Build & Test on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
```

Comment:
This should be fine, but the legacy build environment is Ubuntu 18.04 based.

Reply:
I know, but this code will be built again (and tested) in the legacy build environment. It might actually be more interesting to run the code with other versions of go, e.g. 1.18 when it's out.

Reply:
Fair